en/src/variables.md

```diff
@@ -1,29 +1,29 @@
 # Variables
 
-### Binding and mutablity
+### Binding and mutability
 1. 🌟 A variable can be used only if it has been initialized.
 ```rust,editable
 
-// fix the error below with least modifying
+// Fix the error below with least amount of modification to the code
 fn main() {
-    let x: i32; // uninitialized but using, ERROR !
-    let y: i32; // uninitialized but also unusing, only warning
+    let x: i32; // Uninitialized but used, will result in an ERROR !
```

Comment:
Since these two are inline comments, we can use a simple grammer here  :)

Reply:
There two = Error and Warning lines

Reply:
It is still a simple grammar from the English reader's prospective. The thing is, English speakers don't use something like "Unusing" and other such words. It sounds and looks abnormal. Unusing itself is a spelling mistake. There is no such word in English language.

Also, it can be distracting for them, causing them to focus on the comments more than the code itself.

Reply:
Thank you very much for your patience : ) You are absolutely right, but what I mean is the latter part of the sentence : `Error` seems enough to represent what we want to explain  , from my perspective, there is no need to expand the `Error` to `will result in an ERROR` 🤣.

Reply:
I understand. Thanks for explaining.

Though I'm rather unsure of how to change that old commit, I am thinking of adding another commit where I fix that. Will that be fine by you?

Reply:
There. Is this to your liking?

Reply:
Nope, I learned from the Rust team's official books :D